201-machine-learning-advanced/azuredeploy.json

```diff
@@ -521,6 +540,47 @@
         "hbiWorkspace": "[parameters('confidential_data')]"
       }
     },
+    {
```

Comment:
A couple things here:
1) remove the duplicate resource definition
2) using conditions only on the properties that differ

Reply:
Hi, Brian, from my understanding on this doc, https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/conditional-resource-deployment, condition is only supported on resource level. Would you please show me an example about how to conditionally add new properties into a resource?

Reply:
addressed